src/kind.js

```diff
@@ -83,6 +83,11 @@ var kind = exports = module.exports = function (props) {
 	utils.forEach(kind.features, function(fn){ fn(ctor, props); });
 	
 	if (name) kindCtors[name] = ctor;
+
+	ctor.kind = function (props) {
```

Comment:
Shouldn't this be part of the static handling? As of this code, it would be creating a new function instance for every _kind_ but they can easily inherit the same method attached to different contexts. It would be static so it would look at `this` as the constructor in question as opposed to a memoized reference. This would also decrease stack frames/context switching when executed. JM2C!